unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -165,6 +165,38 @@ public UTF8String toLowerCase() {
     return UTF8String.fromString(toString().toLowerCase());
   }
 
+  public UTF8String initCap(final byte[] byteArr) {
+    String isoString = null;
+    try {
+      isoString = new String(byteArr, "US-ASCII");
```

Comment:
But what about greece characters like `ε`. I guess for handling this special characters we should rely on an implementation from Java/Scala itself.